awesomeBook.js

```diff
@@ -0,0 +1,83 @@
+/** @format */
+
+const bookList = document.querySelector('.booklist');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const addBtn = document.querySelector('#addBtn');
+
+// book
+class Book {
+  constructor() {
+    this.userBook = [];
+  }
+
+  // Get data from localstorage
+  getBook() {
+    this.userBook = JSON.parse(localStorage.getItem('userBook')) || [];
+  }
+
+  // set data to localstorage
+  setBook() {
+    localStorage.setItem('userBook', JSON.stringify(this.userBook));
+  }
+
+  addBooks(title, author) {
+    this.userBook.push({
+      id: Math.floor(Math.random() * 1000),
+      title,
+      author,
+    });
+  }
+
+  showList() {
+    let content = '';
+    this.userBook.forEach((book) => {
+      content += `
+<div
+class="container bg-light d-flex justify-content-between border border-dark p-2 w-50">
+<div>
+<p class="text-dark">"${book.title}" by ${book.author}</p>
+</div>
+<div>
+<button type="submit" class="btn btn-danger removeBtn" id="${book.id}">
+Remove
+</button>
+</div>
+</div>
+    `;
+    });
+
+    return content;
+  }
```

Comment:
- [ ] Good job on the add and remove functionalities, adding a book and removing one is working fine as expected but when the page is reloaded, no book is shown on the page. Kindly work on showing the books even when the page is refreshed 👍🏽